### DISCUSSION TITLE
Is Lit/Web Components/Vanilla supported?

### DISCUSSION THREAD

**Discussion by heydong1** [histoire-dev/histoire#123] [Q&A]
Hi, I used to test my web components written by [Lit](lit.dev) with Storybook, but it's something wrong with its vite builder, I haven't tried histoire with web components, can you give me an answer?

**Comment by Akryum** [comment]
Didn't try it 🤷‍♂️

**Reply by fgeierst** [reply]
Has anyone tried using Lit with Histoire it in the meantime?

**Reply by alantriesagain** [reply]
I just tried for half an hour without much luck. What I did is to create an empty svelte component and a story for it, I can load my lit components with some fixes to not register the component twice but not much works out of the box.
